io.github.wanglin2.mind-map.yml

```diff
@@ -0,0 +1,43 @@
+id: io.github.wanglin2.mind-map
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: run.sh
+rename-desktop-file: thoughts.desktop
+rename-icon: thoughts
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
```

Comment:
what is network for?

Reply:
Version 0.13.1 introduces AI functionality, which requires network access to connect to the API.